src/bun.js/webcore.zig

```diff
@@ -13,6 +14,279 @@ pub const Lifetime = enum {
     temporary,
 };
 
+pub const Alert = struct {
+    pub const Class = JSC.NewClass(
+        void,
+        .{ .name = "alert" },
+        .{
+            .@"call" = .{ .rfn = call },
+        },
+        .{},
+    );
+
+    /// https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-alert
+    pub fn call(
+        // this
+        _: void,
+        ctx: JSC.C.JSContextRef,
+        // function
+        _: JSC.C.JSObjectRef,
+        // thisObject
+        _: JSC.C.JSObjectRef,
+        arguments: []const JSC.C.JSValueRef,
+        _: JSC.C.ExceptionRef,
+    ) JSC.C.JSValueRef {
+        var stdout = std.io.getStdOut();
```

Comment:
Instead of directly using `std.io`, use `Output.writer()` and then call `Output.flush()` at the end. We might also want to add a newline before, but I'm not exactly sure

This will buffer stdout and prevent jumbled text incase stdout could have previously been written to but not flushed yet